recipe/meta.yaml

```diff
@@ -60,4 +58,5 @@ extra:
   recipe-maintainers:
     - alimanfoo
     - jakirkham
+    - joshmoore
```

Comment:
@joshmoore, I added you as a maintainer here. Please let me know whether that is ok 🙂

Reply:
s'ok. ;) cheers